packages/neo-one-smart-contract-lib/src/ownership/Ownable.ts

```diff
@@ -0,0 +1,52 @@
+import { Address, constant, createEventNotifier, SmartContract } from '@neo-one/smart-contract';
+
+/**
+ * @title Ownable
+ * @dev A ownable contract has an owner address
+ */
+
+/* tslint:disable-next-line: variable-name */
+const transferred_ownership = createEventNotifier<Address | undefined, Address>('transferred ownership', 'from', 'to');
+
+export function Ownable<TBase extends Constructor<SmartContract>>(base: TBase) {
+  class OwnableClass extends base {
+    protected mutableOwner: Address | undefined;
+    public constructor(...args: any[]) {
+      super();
+      this.mutableOwner = args[0];
+    }
+
+    @constant
+    public get owner(): Address | undefined {
+      return this.mutableOwner !== undefined ? this.mutableOwner : undefined;
+    }
+
+    @constant
+    public onlyOwner() {
```

Comment:
`protected`, it doesn't need to be accessible outside of the class.

Reply:
👍